disk-utils/fdisk.c

```diff
@@ -263,7 +263,7 @@ static int ask_number(struct fdisk_context *cxt,
 					_("%s (%"PRIu64"-%"PRIu64", default %"PRIu64"): "),
 					q, low, high, dflt);
 	} else if (inchar)
-		snprintf(prompt, sizeof(prompt), _("%s (%c-%c): "),
+		snprintf(prompt, sizeof(prompt), "%s (%c-%c): ",
 				q, tochar(low), tochar(high));
```

Comment:
I'm not sure about it. I can imagine language where people do not use ':'  or '-' etc.

Reply:
That change is reverted in the latest update.